typescript/web/src/utils/tests/data.fixtures.ts

```diff
@@ -0,0 +1,69 @@
+import { Workspace, Dataset, LabelClass } from "@labelflow/graphql-types";
+
```

Comment:
All words in the names defined in this file are inverted.

i.e. `MockDataset` should be `DatasetMock`, `MOCK_WORKSPACE_SIMPLE` should be `SIMPLE_WORKSPACE_MOCK`, etc.

Reply:
I disagree on that, see how the word is generally used in English here:
https://www.wordreference.com/enfr/mock
https://sentence.yourdictionary.com/mock

Reply:
OK I see. Thanks for the links.  
What I've asked you initially was to respect the grammar rule we use with every other variables (even the `APOLLO_MOCKS` constant is named like that). Maybe it's not the most common usage for this exact word but it's the one we use everywhere else except when we need a prefixes like the `NEXT_PUBLIC_` environment variables.
In that sense, and to respect both this rule and the links that you've given me, I think that we should use the `mockery` word if we really can't use mock as an adjective.

i.e. `MockDataset` should be `DatasetMockery`, `MOCK_WORKSPACE_SIMPLE` should be `SIMPLE_WORKSPACE_MOCKERY`, etc.

Reply:
About the word `simple`, shouldn't it be `single` instead?

Reply:
`simple` here means that there is nothing fancy, for example here `MOCK_DATASET_SIMPLE` doesn't have label classes, compared to `MOCK_DATASET_WITH_CLASSES`.

Reply:
OK, thanks for the clarification 👍  
Simple is OK then.
If you want to disambiguate the things a little bit more when it's necessary to differentiate such variants, you can use the words `minimal` and `exhaustive`. But at this point, it's not that important...
e.g. `MINIMAL_WORKSPACE_MOCKERY` 🤷‍♂️